src/app/screens/BookDetail/layout.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import BookAttr from './components/BookAttr/';
+import Suggestions from './components/Suggestions/';
+import NewComment from './components/NewComment/';
+import Comments from './components/Comments/';
+
+function BookDet({ book }) {
+  return (
+    <div>
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
check this comment to solve the `eslint` problem

https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/340#issuecomment-338424908